tests/conftest.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+from typing import Iterator
+
+import pytest
+
+
+@pytest.fixture
+def tmp_file(mode: str = "w") -> Iterator[Path]:
+    fp = Path(NamedTemporaryFile(mode).name)
+    try:
+        yield fp
+    finally:
+        fp.unlink()
```

Comment:
Can it fail if the file doesn't exist?

Reply:
Eagle-eye!
Added `missing_ok=True` in a0dbccb4abfae